api/app/models.py

```diff
@@ -62,7 +61,9 @@ class Bucketlist(db.Model):
                              default=db.func.current_timestamp())
     date_modified = db.Column(db.DateTime,
                               default=db.func.current_timestamp())
-    created_by = db.Column(db.String(20))
+    created_by = db.Column(db.Integer, db.ForeignKey('user.id'))
+    user_id = db.relationship('User', backref='bucketlist')
```

Comment:
**Code Quality: Nitpick**

Not sure why this is here though. Is this not serving the same purpose as `created_by`